src/components/TodoList.jsx

```diff
@@ -0,0 +1,15 @@
+const todoList = [
+  { id: 1, title: 'Learn about CRA way to initiate React app' },
+  { id: 2, title: 'Learn about Vite'},
+  { id: 3, title: 'Try creating app using CRA' },
+  { id: 4, title: 'Try creating app using Vite' },
+  { id: 4, title: 'Keep developing using better option' },
```

Comment:
NON-BLOCKING: You'll want to make this id unique (you have to objects with `id = 4`).

Reply:
I caught this one too and fixed it in my next PR.